packages/data/addon/services/navi-facts.ts

```diff
@@ -70,8 +70,7 @@ export default class NaviFactsService extends Service {
    */
   fetch(this: NaviFactsService, request: RequestV1, options: RequestOptions): Promise<NaviFactsModel> {
     const dataSource = options?.dataSourceName || getDefaultDataSourceName();
-    const type = config.navi.dataSources?.find((source: { name: string; type: string }) => source.name === dataSource)
-      ?.type;
+    const type = getDataSourceField('type', dataSource);
```

Comment:
Do we need a special function for this? Can, we just get the default data source and then pull out the field?